src/vs/platform/actions/browser/menuEntryActionViewItem.ts

```diff
@@ -304,8 +304,8 @@ export class LabeledMenuItemActionItem extends MenuEntryActionViewItem {
 					iconClass = ICON_PATH_TO_CSS_RULES.get(iconPathMapKey)!;
 				} else {
 					iconClass = ids.nextId();
-					createCSSRule(`.codicon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.light || item.icon.dark)}`);
-					createCSSRule(`.vs-dark .codicon.${iconClass}, .hc-black .codicon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.dark)}`);
+					createCSSRule(`.codicon.${iconClass}`, `background-image: ${asCSSUrl(item.icon.light || item.icon.dark)} !important`);
```

Comment:
could you add the // {{SQL CARBON EDIT}} tags?  They are still useful during the code merge process to understand why there are merge conflicts.

Reply:
Alright, cleaned it up. No logic changed - just moved the two custom classes into our own code.